packages/unit-testing/jest-configuration/SuiteCloudJestConfiguration.js

```diff
@@ -181,6 +189,10 @@ const CORE_STUBS = [
 		module: 'N/https',
 		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/https.js`,
 	},
+	{
+		module: 'N/https/clientCertificate',
+		path: `<rootDir>/node_modules/${CORE_STUBS_PATH}/https/clientCertificate.js`,
+	},
```

Comment:
Remove this one, we have moved it into `/https/clientCertificate/clientCertificate.js` and it's arleady defined at lines 208-211.